bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/VerisureSession.java

```diff
@@ -489,9 +528,21 @@ private synchronized boolean logIn() {
         try {
             if (!areWeLoggedIn()) {
                 logger.debug("Attempting to log in to mypages.verisure.com");
-                String url = LOGON_SUF;
+                logger.debug("Remove all cookies");
+                CookieStore cookieStore = httpClient.getCookieStore();
+                cookieStore.removeAll();
```

Comment:
You are using the common HTTP client if I am not wrong, so removing all cookies looks like a bad idea.

Reply:
Searching in other bindings, I discovered the MyQ binding is doing the same "error".

I guess you just have to remove your specific cookies rather than all.

Reply:
See PR #11343  how it was fixed.